easybuild/easyconfigs/m/MUSCLE/MUSCLE-5.0.1428-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MakeCp'
+
+name = 'MUSCLE'
+version = '5.0.1428'
+
+homepage = 'https://drive5.com/muscle/'
+description = """MUSCLE is one of the best-performing multiple alignment programs
+ according to published benchmark tests, with accuracy and speed that are consistently
+ better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users
+ learn everything they need to know about MUSCLE in a few minutes-only a handful of
+ command-line options are needed to perform common alignment tasks."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'rcedgar'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_add-ver-counter-header.patch']
+checksums = [
+    '59a8196890893e44a60f24134e8da24ee8381830a0f84e37378a3ca8be5430f5',  # v5.0.1428.tar.gz
+    '129e3cd5cd7ae4d9819bbb36776275781e33c82410ee1e5d480b3de2df213931',  # MUSCLE-5.0.1428_add-ver-counter-header.patch
+]
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('ccache', '4.2.1', '', True),
```

Comment:
@lexming `ccache` can not be installed on bare metal and IoTs as `glibc-static` and `libc6-dev` are required, but I guess `GCC` v10 is way advanced for that app anyway. In summary, happy for this PR as most applications are HPCs. Thx @lexming